installers/msi-language/StateDeploy/CustomAction.cs

```diff
@@ -563,7 +644,13 @@ public static ActionResult GAReportStart(Session session)
         public static ActionResult GAReportUserExit(Session session)
         {
             session.Log("sending user exit event");
-            TrackerSingleton.Instance.TrackEventSynchronously(session, session["SESSION_ID"], "stage", "finished", "cancelled", session["ProductVersion"]);
+            var sessionID = GetSessionIDForExitAction(session);
+            if (sessionID == "unset")
+            {
+                // This can happen, when the user cancelled on the Welcome Dialog, before the session id has been generated.
+                sessionID = NewSessionID(session, session["ProductVersion"]);
```

Comment:
This is another instance where the started event may be reported twice

Reply:
Same as above...